Data/Aeson.hs

```diff
@@ -274,11 +274,9 @@ eitherDecodeStrict' =
 -- > {-# LANGUAGE OverloadedStrings #-}
 -- >
 -- > instance FromJSON Person where
--- >     parseJSON (Object v) = Person <$>
--- >                            v .: "name" <*>
--- >                            v .: "age"
--- >     -- A non-Object value is of the wrong type, so fail.
--- >     parseJSON _          = empty
+-- >   parseJSON = withObject "Person" $ \v -> Person
```

Comment:
Most code examples use 4 space indents. It's worth to keep the consistent style.